settings.py

```diff
@@ -745,10 +745,18 @@
 }
 
 LOCAL_AVAILABLE_CUSTOM_RULE_CRITERIA = {}
-AVAILABLE_CUSTOM_RULE_CRITERIA = {}
+AVAILABLE_CUSTOM_RULE_CRITERIA = {
+    'COVID_US_ASSOCIATED_USER_CASES':
+        ['custom.covid.rules.custom_criteria.associated_usercase_closed',
+         "Custom: filter checkin cases that are open, associated to a mobile worker, and usercase is closed"]
```

Comment:
What's the description for? [This line](https://github.com/dimagi/commcare-hq/blob/c99d919da8e76585e21eb4d4073cf00cec2e8eac/corehq/apps/data_interfaces/models.py#L614) makes me think that each value in `AVAILABLE_CUSTOM_RULE_CRITERIA` should be just a function path string, not a list.